File: CrimsonClubs/CrimsonClubs.Models/Dtos/AddClubDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CrimsonClubs.Models.Dtos
{
    public class AddClubDto
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public bool IsRequestToJoin { get; set; }
        public int? GroupId { get; set; }
    }
}

File: Android/CrimsonClubs/app/src/main/java/crimsonclubs/uacs/android/crimsonclubs/AddClubDto.java
package crimsonclubs.uacs.android.crimsonclubs;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AddClubDto {

    @Expose
    @SerializedName("name")
    public String name;
    @Expose
    @SerializedName("description")
    public String description;
    @Expose
    @SerializedName("isRequestToJoin")
    public boolean isRequestToJoin;
    @Expose
    @SerializedName("groupId")
    public int groupId;

}
